pkg/apps/controller/deployer/deployer_controller.go

```diff
@@ -420,6 +421,7 @@ func (c *DeploymentController) makeDeployerPod(deployment *v1.ReplicationControl
 			RestartPolicy:                 v1.RestartPolicyNever,
 			ServiceAccountName:            c.serviceAccount,
 			TerminationGracePeriodSeconds: &gracePeriod,
+			ShareProcessNamespace:         &shareProcessNamespace,
```

Comment:
Why is this a thing?  Is the default bad somehow?

Reply:
Also the deployer pod is single-container Pod... From what I saw in kube, this is still an alpha feature that is disabled by default anyway ?

Reply:
This makes the default explicit, had to set something or  explicitly ignore the field in the unit test, which I didn't want to do